### PR TITLE
Makefile and README updates for sdk 1.0

### LINKED ISSUE #169: Update README after Operator SDK upgrade
The Makefile targets and workflow for local development have changed with #153. We need to update the README with revised information.

### PR DESCRIPTION
**Pull request by andrewazores**
Fixes #169

### CODE REVIEW

**Comment by andrewazores** [comment]
`make deploy` is broken currently, it will only try to deploy into the `container-jfr-operator-system` namespace and not the current namespace (`oc project -q`). I'm trying to figure out how to get that to work. The README says it does work in the meantime, anyway.

File: README.md [hunk 1]
```diff
@@ -8,68 +8,58 @@ API to manage [JDK Flight Recordings](https://openjdk.java.net/jeps/328).
...
-OpenShift console by first adding a custom catalog source. A YAML definition
-of such a custom source is at `deploy/olm-catalog/catalog-source.yaml`, which
-can be added to your cluster by
-`oc create -f deploy/olm-catalog/catalog-source.yaml` or simply `make catalog`.
-This method allows the latest released version of the operator to be installed
-into the cluster with the same method and in the same form that end users would
-receive it.
+The operator can be deployed using OLM using `make deploy_bundle`. This will
+deploy `quay.io/rh-jmc-team/container-jfr-operator-bundle:$IMAGE_VERSION` to
+your configured cluster using `oc`. You can set the variables `IMAGE_NAMESPACE`
```
Comment (ebaron) [inline README.md:59]: This should probably be `kubectl` to be consistent with the above text.

**Review by ebaron — COMMENTED** [review]
Looks good Andrew. Just a few points below.

File: README.md (hunk 1 above)
Comment (andrewazores) [reply README.md]: This ends up calling `operator-sdk run bundle`, and I'm not sure if this does call `oc`/`kubectl` or what it does exactly, so I've updated the README to be a little less specific here and cover oc/kubectl/kubeconfig...

**Review by ebaron — APPROVED** [review]
LGTM. Thanks for doing this!

**Pull request merged**